### PR TITLE
Graphite 1.1 Tags

### PR DESCRIPTION
**Pull request by cody271**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,17 +25,19 @@ $ sudo python setup.py install --with-flatcc=PREFIX
...
+The use of `--pure-python` is provided for convenience; However, the native C module is recommended for best performance.
 
 Using with graphite-web
 -----------------------
 
 In your graphite's `local_settings.py`:
 
     STORAGE_FINDERS = (
-        'irondb.IronDBFinder',
+        'irondb.IRONdbFinder',
```
Comment (rileyberton) [inline README.md]: I don't think we can change this.  This will break all existing installs if they upgrade.

File: irondb/irondb.py [hunk 2]
```diff
@@ -144,73 +218,29 @@ def get_intervals(self):
         return IntervalSet([Interval(0, int(time.time()))])
 
 
-class IronDBFinder(BaseFinder):
+class IRONdbFinder(BaseFinder):
```
Comment (rileyberton) [inline irondb/irondb.py]: Changing this name is going to break existing installs.  Can we add an alias class to cover both names for backwards compatibility?

File: irondb/irondb.py [hunk 3]
```diff
@@ -313,16 +313,18 @@ def find_nodes(self, query):
...
         for name in names:
             if 'leaf' in name and 'leaf_data' in name:
                 fetcher.add_leaf(name['name'], name['leaf_data'])
-                reader = IronDBReader(name['name'], fetcher)
+                reader = IRONdbReader(name['name'], fetcher)
                 yield LeafNode(name['name'], reader)
             else:
                 yield BranchNode(name['name'])
+
+IronDBFinder = IRONdbFinder
```
Comment (cody271) [inline irondb/irondb.py]: The alias to the old name is here.

Comment (rileyberton) [reply irondb/irondb.py]: I must have missed it, my bad

**Pull request merged**